packages/oracles-starter-kit/contracts/supra-oracles/SupraValueFeedExample.sol

```diff
@@ -0,0 +1,21 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.7;
+
+interface ISupraSValueFeed {
+    function checkPrice(
+        string memory marketPair
+    ) external view returns (int256 price, uint256 timestamp);
+}
+
+contract SupraValueFeedExample {
+    ISupraSValueFeed internal sValueFeed;
+
+    constructor() {
+        sValueFeed = ISupraSValueFeed(0x7f003178060af3904b8b70fEa066AEE28e85043E);
+    }
+
+    function getPrice() external view returns (int) {
+        (int price /* uint timestamp */, ) = sValueFeed.checkPrice("btc_usdt");
```

Comment:
can we externalize btc_usdt ? And also we can provide https://supraoracles.com/docs/get-started/market-pairs#klaytn-chain and https://supraoracles.com/docs/get-started/networks as references from where the values are taken

Reply:
Sure, will add